### ISSUE TITLE
Seamless mode

### ISSUE THREAD

**Issue by stuaxo** [tonioni/WinUAE#98]
JanusUAE (just for morphOS) has seamless mode.  It would be great if this was available on other platforms so amiga apps such as deluxe paint could be ran as normal apps.

**Comment by tonioni** [comment]
It would be nice but..

It is not that complex or difficult when host OS is 1:1 compatible with AmigaOS but it is totally different task with non-AmigaOS host OS. Another problem is Amiga RTG system which is 100% undocumented secret.

**Comment by timofonic** [comment]
Fixable? Feature request for the far future? Won't fix? 

Any news? :)

**Comment by stuaxo** [comment]
I wonder if any information can be had by looking at the open source amiga graphics card project by  @mntmn since he had to write a picasso96 driver?

https://github.com/mntmn/amiga2000-gfxcard#picasso96-writing-a-driver-for-a-closed-system

**Comment by tonioni** [comment]
No. Picasso96 SDK has been available since ages ago (It was licensed by Cloanto). It is useless for anything else than making Picasso96 drivers. All the magic is in Picasso96 core.

**Comment by stuaxo** [comment]
I wonder if the RTG information is more likely to emerge now that some of Amigas legal issues are finally being settled, there seems to be a willingness to look forward project-wise.

(At least that's what I got from this Amiga Bill episode with Mike Battilana on Amigas future)
https://www.youtube.com/watch?v=lp-NAajrc5Q

Note for myself in the future when I come across this issue again:
I think if I look at this again, I should probably start from the Janus UAE end and try and understand how that works before returning here.

**Comment by tonioni** [comment]
That is possible. Lets wait and see. But in other topic than this one.
